gloss/crt0.S

```diff
@@ -260,6 +260,9 @@ _skip_init:
   csrr a5, mstatus
   ori a5, a5, 0x200
   csrw mstatus, a5
+  csrwi vstart, 0
+  csrwi vxrm, 0
+  csrwi vxsat, 0
```

Comment:
Use `csrwi vcsr, 0` here, which is smaller and more future-proof (it will initialize other random bits of state that get added down the road).

Reply:
Thanks!